rancher/latest/en/kubernetes/ingress/index.md

```diff
@@ -308,3 +304,122 @@ The `address` in the ingress will be the public endpoint of where the load balan
 
 To access your application, you can hit the address at our default port `80` (i.e. `http://1.2.3.4:80`) or the address directly (i.e. `http://1.2.3.4`). 
 
+### Load Balancer Options with Kubernetes Ingress 
+
+By default, a Kubernetes ingress will deploy 1 load balancer on only 1 host using port `80`. Rancher has added the ability to support multiple load balancers using the port of your choice. By scaling the ingress, the address programmed in Kubernetes will also reflect all the hosts that have the load balancer available. 
+
+> **Note:** If you choose to increase the scale of your ingress, you will need to ensure that there are at least the equivalent number of hosts available in your Kubernetes environment that have the port available. 
+
+#### Setting up a Simple Service
+
+Before setting up any ingress, services will need to be created in Kubernetes. We'll start by adding a service and replication controller into our Kubernetes environment. 
+
+Here's a simple nginx service to add into Kubernetes.
+
+Example `nginx-service.yml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-service
+  labels:
+    k8s-app: nginx-service
+spec:
+  ports:
+  - port: 90
+    targetPort: 80
+    protocol: TCP
+    name: http
+  selector:
+    k8s-app: nginx-service
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: nginx-service
+spec:
+  replicas: 1
+  selector:
+    k8s-app: nginx-service
+  template:
+    metadata:
+      labels:
+        k8s-app: nginx-service
+    spec:
+      terminationGracePeriodSeconds: 60
+      containers:
+      - name: nginx-service
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+```
+
+Using `kubectl`, let's launch our nginx service into Kubernetes. Remember, you can either [configure `kubectl` for your local machine]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/kubernetes/#kubectl) or you can use the shell in the UI under **Kubernetes** -> **kubectl**.
+
+```bash
+$ kubectl create -f nginx-service.yml
+service "nginx-service" created
+replicationcontroller "nginx-service" created
+```
+
+#### Example of 2 Load Balancers using an Alternative Port
+
+You can set up a ingress that is launched using 2 load balancers on 2 different hosts that uses a port, `99`, instead of the default ingress port `80`. For this ingress to work correctly, your kubernetes environment will need at least 2 hosts that have port `99` available. 
+
+Example `scaled-ingress.yml`
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: scaledlb
+  annotations:
+    scale: "2"
+    http.port: "99"
+spec:
+  backend:
+    serviceName: nginx-service
+    servicePort: 90
+```
+
+Let's create the ingress using `kubectl`. After you create the ingress, the ingress controller will trigger 2 load balancers to be created and visible in the **kubernetes-ingress-lbs** stack within the **Kubernetes** -> **System** tab. 
```

Comment:
2 load balancers = 1 lb service with 2 instances. May be we should emphasize that